05libraries/sec00Intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Libraries
+---
+
+## Why use Libraries?
+
+> The best code is the code you never write
+
+### What are libraries?
+
+- Libraries are collections of useful classes and functions, ready to use
+- C++ libraries can be somewhat harder to use than modules in other languages (e.g. Python)
+- Can save time and effort by providing well-tested, flexible, optimised features
```

Comment:
Some explanation of why we put code into libraries, and how they fit into a project (or projects) would be useful, how it differs from a standard executable, and maybe something about static and dynamic linking since it comes up in Section 1 (choosing libraries) before any discussion of what that means happens.